google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -951,7 +951,14 @@ def download path = nil, verify: :md5, encryption_key: nil, range: nil,
         end
 
         ##
-        # Copy the file to a new location.
+        # Copies the file to a new location. Metadata from the source object will
+        # be copied to the destination object unless a block is provided.
+        #
+        # If the optional block for updating metadata fields is provided, only the
+        # updates made in this block will appear in the destination object, and
+        # other metadata fields in the destination object will be empty. To copy
+        # all source object metadata while updating destination object fields in a
+        # block, use the `force_copy_metadata: true` flag.
```

Comment:
Add: 
> use the `force_copy_metadata: true` flag, and the library will copy metadata from source metadata into the copy request.

I want the user to understand the library is doing it rather than the API.

Reply:
Will do.